control/stabilisation_copter_default_config.h

```diff
@@ -526,6 +525,483 @@ stabilisation_copter_conf_t stabilisation_copter_default_config =
 	}
 };
 
+static stabilisation_copter_conf_t stabilisation_copter_custom_config =
```

Comment:
I thought we were using the default gain, then no need to keep those one on the clean version of your code